Automattic/wp-calypso
# `SitePreview`: refactor away from `UNSAFE_*`

#### Changes proposed in this Pull Request

* `SitePreview`: refactor away from `UNSAFE_*`

#### Testing instructions

* Go to `/pages/:site` (or `/posts/:site`)
* Choose a page (or post) and hit the three-dots menu, click _View_
* Verify the preview is still working as expected

To further test the changes, inspect the `previewUrl` passed on to the `<WebPreview />` component. The `cachebust` param should increment by 1 every time you preview a page (or post). If you change your currently active site, it should reset.

Related to #58453
